tsdb/engine/tsm1/compact_test.go

```diff
@@ -2640,17 +2521,6 @@ func TestDefaultPlanner_PlanOptimize_Test(t *testing.T) {
 
 			cp := tsm1.NewDefaultPlanner(ffs, tsdb.DefaultCompactFullWriteColdDuration)
 			expectedNotFullyCompacted(cp, test.expectedFullyCompactedReasonExp, test.expectedgenerationCount)
-
-			// Reverse test files and re-run tests
-			slices.Reverse(test.fs)
```

Comment:
It is not realistic to have files with smaller block sizes come before files with larger block sizes in a single generation.